docs/user/howto_test_offline.md

```diff
@@ -0,0 +1,126 @@
+# Running MicroShift Fully Offline
+
+There are several steps needed to run MicroShift in a fully offline
+setting, with no network access at all. This document describes a
+process for setting up a host to test MicroShift with no network.
+
+## Image Management
+
+Refer to [Embedding MicroShift Container Images for Offline
+Deployments](howto_offline_containers.md) for instructions for
+preparing the ostree image with the necessary container images
+embedded when they cannot be pulled over the network.
+
+## MicroShift Configuration
+
+Kubernetes expects to have some basic network settings on the host for
+node identification and DNS. When real settings are not going to be
+available, alternatives need to be provided for robust disconnected
+operation.
+
+1. Start by preparing a VM or host in the normal way and installing
+   the version of MicroShift you want to test.
+
+2. Stop MicroShift and clean up the OVN data.
+
+```
+systemctl stop microshift
+systemctl stop kubepods.slice
+/usr/bin/microshift-cleanup-data --ovn
+```
+
+3. Determine the interface on the host that has the default route
+
+```
+IFACE=$(ip route | grep default | awk '{print $5}')
+```
+
+(This is usually enp1s0 in a VM)
+
+4. Stop external network access by bringing the NIC down.
+
+```
+nmcli connection down $IFACE
+nmcli connection modify $IFACE connection.autoconnect no
+```
+
+5. Ensure the host has a persistent name
+
+```
+hostnamectl hostname $NAME
```

Comment:
Shoud we set `NAME` and `IP` variables here as they are used a few times below?

Reply:
I'll add some text to explain what they are.